app-emulation/qemu/files/qemu-7.2.0-disable-gmp.patch

```diff
@@ -0,0 +1,45 @@
+commit 433c2acd1b9c3f7f5903273e85fed9a7f9157b23
+Author: John Helmert III <ajak@gentoo.org>
+Date:   Wed Jan 4 12:25:08 2023 -0600
+
+    meson.build: disable gmp
+    
+    gmp isn't required, and we want to avoid linking to it automatigcally
+    in Gentoo.
+    
+    Bug: https://bugs.gentoo.org/886207
+    Signed-off-by: John Helmert III <ajak@gentoo.org>
+
+diff --git a/meson.build b/meson.build
+index 5c6b5a1c75..2f9550362d 100644
+--- a/meson.build
++++ b/meson.build
+@@ -1234,15 +1234,6 @@ if not gnutls_crypto.found()
+   endif
+ endif
+ 
+-gmp = dependency('gmp', required: false, method: 'pkg-config', kwargs: static_kwargs)
```

Comment:
Does this break something else which then refers to gmp?

Reply:
I don't think so. The build log contains no references to it, even with gmp still installed:

```
# emerge -qpv gmp
[ebuild   R   ] dev-libs/gmp-6.2.1-r2  USE="asm cxx -doc -pic -static-libs" ABI_X86="(64) -32 (-x32)"
# grep gmp /var/tmp/portage/app-emulation/qemu-7.2.0-r1/temp/build.log
 * Applying qemu-7.2.0-disable-gmp.patch ...
```

I also don't see any references to any gmp .so or header in the build log or even in the source.